aws/basic-eks-cluster/blueprint.yaml

```diff
@@ -66,6 +67,7 @@ spec:
   - name: NodeInstanceType
     type: Select
     description: What EC2 instance type do you want to use for the nodes?
+    default: t2.xlarge
```

Comment:
This is quite big to be the default. IMO we should have `t2.medium` as default.

Reply:
we help aws to make money

Reply:
stop giving them money